### PR TITLE
feat: add example for CRD Datasource test

### PR DESCRIPTION
**Pull request by zqq454224016**
add example

### CODE REVIEW

File: config/rbac/role_binding.yaml [hunk 1]
```diff
@@ -10,3 +10,17 @@ subjects:
...
+
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: manager-rolebinding-cluster-admin
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: cluster-admin
```
Comment (bjwswang) [inline config/rbac/role_binding.yaml:22]: @zqq454224016 our arcadia do not require `cluster-admin` . Please minimize this to the least permission

File: config/samples/arcadia_v1alpha1_datasource.yaml [hunk 2]
```diff
@@ -2,7 +2,7 @@ apiVersion: arcadia.kubeagi.k8s.com.cn/v1alpha1
 kind: Datasource
 metadata:
   name: datasource-sample
-  namespace: kubebb-system
+  namespace: arcadia-system
 spec:
-  url: my-minio.kubebb-system.svc.cluster.local:9000
+  url: my-minio.arcadia-system.svc.cluster.local:9000
```
Comment (bjwswang) [inline config/samples/arcadia_v1alpha1_datasource.yaml:7]: hi @zqq454224016  use namespace `arcadia` is fine. do not need to use `arcadia-system`

**Comment by bjwswang** [comment]
@zqq454224016  Good job!  Please resolve the above comments.

File: config/samples/arcadia_v1alpha1_datasource.yaml (hunk 2 above)
Comment (zqq454224016) [reply config/samples/arcadia_v1alpha1_datasource.yaml:7]: Because arcadia has been deployed in this namespace

Comment (bjwswang) [reply config/samples/arcadia_v1alpha1_datasource.yaml:7]: You mean `minio` ? We all use same  namespace `arcadia`.If it is not,pls change it.

File: config/rbac/role_binding.yaml (hunk 1 above)
Comment (zqq454224016) [reply config/rbac/role_binding.yaml:22]: manager-role does not have sufficient permissions

Comment (bjwswang) [reply config/rbac/role_binding.yaml:22]: then you should add the missing permissions to manager-role.

File: config/samples/example-test.sh [hunk 3]
```diff
@@ -0,0 +1,193 @@
...
+	done
+}
+info "1. create kind cluster"
+make kind
+
+info "2. install kubebb core"
+info "2.1 deploy kubebb/core and minio"
+helm repo add kubebb https://kubebb.github.io/components/
+helm repo update
+helm install -n arcadia --create-namespace kubebb-core kubebb/kubebb-core
```
Comment (bjwswang) [inline config/samples/example-test.sh:179]: Seems you are trying to install kubebb which should be in namespace kubebb-system

File: config/samples/example-test.sh [hunk 4]
```diff
@@ -0,0 +1,193 @@
...
+helm install my-minio -n arcadia kubebb/minio
+waitPodReady "arcadia" "release=my-minio"
+
+info "3. install arcadia"
+docker tag controller:latest controller:example-e2e
+kind load docker-image controller:example-e2e --name=$KindName
+make deploy IMG="controller:example-e2e"
+kubectl wait deploy -n arcadia arcadia-controller-manager --for condition=Available=True --timeout=$Timeout
```
Comment (bjwswang) [inline config/samples/example-test.sh:187]: where is the `make install` ?

File: controllers/datasource_controller.go [hunk 5]
```diff
@@ -49,6 +49,7 @@ type DatasourceReconciler struct {
 //+kubebuilder:rbac:groups=arcadia.kubeagi.k8s.com.cn,resources=datasources,verbs=get;list;watch;create;update;patch;delete
 //+kubebuilder:rbac:groups=arcadia.kubeagi.k8s.com.cn,resources=datasources/status,verbs=get;update;patch
 //+kubebuilder:rbac:groups=arcadia.kubeagi.k8s.com.cn,resources=datasources/finalizers,verbs=update
+//+kubebuilder:rbac:groups=arcadia,resources=secrets,verbs=get;list
```
Comment (bjwswang) [inline controllers/datasource_controller.go:52]: @zqq454224016  You have used a wrong group .Should be "" or

**Review by bjwswang — APPROVED** [review]

**Pull request merged**